### ISSUE TITLE
Expose camera ports

### ISSUE THREAD

**Issue by mdurrani808** [PhotonVision/photonvision#557]
Right now our solution for people to get their camera ports is to open their stream in a new tab, this isn't ideal. It should be displayed in the ui or be the alt text for the stream if possible.

**Comment by mdurrani808** [comment]
I see now with thin client they just click on it and the port is listed. Good enough for me.
